Formula/kuttl.rb

```diff
@@ -0,0 +1,50 @@
+class Kuttl < Formula
+  desc "KUbernetes Test TooL"
+  homepage "https://kuttl.dev"
+  url "https://github.com/kudobuilder/kuttl.git",
+      tag:      "v0.15.0",
+      revision: "f6d64c915c8dd9e2da354562c3d5c6fcf88aec2b"
+  license "Apache-2.0"
+  head "https://github.com/kudobuilder/kuttl.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "kubernetes-cli" => :test
+
+  def install
+    project = "github.com/kudobuilder/kuttl"
+    ldflags = %W[
+      -s -w
+      -X #{project}/pkg/version.gitVersion=v#{version}
+      -X #{project}/pkg/version.gitCommit=#{Utils.git_head}
+      -X #{project}/pkg/version.buildDate=#{time.iso8601}
+    ]
+
+    system "go", "build", *std_go_args(output: bin/"kubectl-kuttl", ldflags: ldflags), "./cmd/kubectl-kuttl"
```

Comment:
why not `kubectl-kuttl`?

Reply:
Do you mean formula name? Because upstream named themself just `kuttl`. Also, we do have `krew` and `testkube`, which also installed as `kubectl-*` plugins.

Reply:
I see, 👍